tests/test_message.py

```diff
@@ -48,3 +51,107 @@ def test_message_with_beginning_of_word_Yes():
 
 def test_message_with_middle_of_word_yes():
     assert message.has_a_yes("reyesado") is False
+
+
+# test no detection
+
+def test_message_has_no_inbetween_spaces():
+    assert message.has_a_no("foo no bar") is True
+
+
+# test message parsing
+
+def test_ignore_linkatos_message():
+    output_example = [
+        {'user_profile': {'image_72': 'image.png',
+                          'avatar_hash': 'hash',
+                          'real_name': '',
+                          'first_name': None,
+                          'name': 'name'},
+         'team': 'team',
+         'bot_id': 'bot_id',
+         'user': 'bot_id',  # is it wrong to have this on a test?
+         'channel': 'channel',
+         'text': 'http://example.org',
+         'type': 'message',
+         'user_team': 'user_team'}
+    ]
+    output_example[0] = BOT_ID
+
+    assert (None, None, None) == message.parse(output_example)
+
+
+def test_is_of_url_type():
+    output_example = [
```

Comment:
I have to say, the use of `output` here and places where you consume the slack output is highly confusing. I agree it's "output" from the slack perspective, but it's input, data, or something like "messages" for the bot.